components/bl-accordion-component/src/index.js

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react';
+
+import { useItemsState } from './helpers';
+import { Item } from './item';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function AccordionComponent({ component, eventHandlers }) {
+  const { classList, display, accordionData, controlledAccordion, style } = component;
+  const { onClick, onMouseOver, onMouseOut } = eventHandlers;
+
+  const [accordionId, setAccordionId] = useState(() => Math.random().toString(16).slice(-8));
```

Comment:
We have an ID generator `BackendlessUI.UUID.short()`
`const accordionId = useMemo(() => BackendlessUI.UUID.short(), []);`